examples/asyncawait.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const fastify = require('../fastify')({ logger: true })
+const fastify = require('../fastify')({ loggerInstance: true })
```

Comment:
I'm not going to go through all 58 of the changed files, but I can tell you that this is very much not correct. `loggerInstance: true` is invalid.